benches/benches/bevy_ecs/change_detection.rs

```diff
@@ -0,0 +1,102 @@
+use bevy_ecs::{
+    component::Component,
+    entity::Entity,
+    prelude::{Added, Changed},
+    world::World,
+};
+use criterion::{black_box, criterion_group, criterion_main, Criterion};
+use rand::{prelude::SliceRandom, SeedableRng};
+use rand_chacha::ChaCha8Rng;
+
+criterion_group!(benches, added_detection, changed_detection,);
+criterion_main!(benches);
+
+#[derive(Component, Default)]
+#[component(storage = "Table")]
+struct Table(f32);
+#[derive(Component, Default)]
+#[component(storage = "SparseSet")]
+struct Sparse(f32);
+
+const RANGE: std::ops::Range<u32> = 5..7;
+
+fn deterministic_rand() -> ChaCha8Rng {
```

Comment:
A quick search for `deterministic_rand` in the PR changeset doesn't bring anything up - what is this used for?

Reply:
It's now used for shuffling the `few_changed_detection` bench entities